pkg/manager/member/tikv_member_manager.go

```diff
@@ -445,6 +445,7 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 	podSpec.SecurityContext = podSecurityContext
 	podSpec.InitContainers = initContainers
 	podSpec.Containers = []corev1.Container{tikvContainer}
+	podSpec.ServiceAccountName = tc.Spec.TiKV.ServiceAccount
```

Comment:
Please make sure that this will not trigger a rolling update during tidb-operator upgrade if `ServiceAccount` not configured.